### PR TITLE
community/tclx: remove use of rresvport

### PR DESCRIPTION
**Pull request by snoe925**
Remove use of network routines in libc on Debian.  In this case rresvport is just a helper to allocate a port < 1024.  The user can do that as root.

### CODE REVIEW

**Comment by jirutka** [comment]
> Remove use of network routines in libc on Debian.

WAT? This is not Debian… o.O

File: community/tclx/APKBUILD [hunk 1]
```diff
@@ -11,11 +11,18 @@ depends="tcl"
...
+	rresvport.patch
+	"
 
 _major=${pkgver%.*}
 builddir="$srcdir"/tclx${_major}
 
+prepare() {
+	cd "$builddir"
+	default_prepare
+}
```
Comment (jirutka) [inline community/tclx/APKBUILD]: Why have you added this? `default_prepare` is called by default when custom `prepare` function is not defined.

Comment (snoe925) [inline community/tclx/APKBUILD]: I used main/openssh as an example.  I cut and pasted the prepare, but after removing one line I did not understand to remove the entire block.

**Comment by fabled** [comment]
Applied with pkgrel bump in commit 1ab30ef138c7b4200ee7c543bafa9502869799a3. I had to merge manually using the diff because this was on top of your earlier tree, and not on top of the aports master. Please submit further changes rebased on top of the aports master branch. Thanks.

**Pull request closed without merging**